docs/sources/whatsnew/whats-new-next/index.md

```diff
@@ -17,6 +17,23 @@ weight: -37
 
 Welcome to Grafana Cloud! Read on to learn about the newest changes to Grafana Cloud.
 
+## Distributed Tracing in Grafana Cloud k6
+
+<!-- Heitor Tashiro Sergent -->
+
+_Generally available in Grafana Cloud_
+
+You can now use the Grafana Cloud Traces integration with Grafana Cloud k6 to quickly debug failed performance tests and proactively improve application reliability.
+
+Distributed tracing in Grafana Cloud k6 only requires two things:
+
+- An application instrumented for tracing with Grafana Cloud Traces.
+- Adding a few lines of code to your existing k6 scripts.
```

Comment:
This is a nit, but for parallel structure, I'd want this to be the noun form. But as I said, a nit and it makes it longer, so take it or leave it.
```suggestion
- The addition of a few lines of code to your existing k6 scripts.
```

Reply:
I'm going to leave this one as it is, I think it "sounds" better this way, even if it'd be more consistent with your suggestion.